pylint/checkers/base.py

```diff
@@ -63,7 +63,7 @@
 # Copyright (c) 2021 Lorena B <46202743+lorena-b@users.noreply.github.com>
 # Copyright (c) 2021 David Liu <david@cs.toronto.edu>
 # Copyright (c) 2021 Andreas Finkler <andi.finkler@gmail.com>
-# Copyright (c) 2021 Or Bahari <orbahari@mail.tau.ac.il>
+# Copyright (c) 2021-2022 Or Bahari <or.ba402@gmail.com>
```

Comment:
You need to modify the copyrite aliases so it's done automatically.

Reply:
I understood so I would update `.copyrite_aliases` and keep those manual changes too?

Reply:
Yes, you can only update the config file, I upgrade all code files automatically during release